schema/deploy/shipping/views.sql

```diff
@@ -1080,7 +1080,9 @@ create or replace view shipping.hcov19_observation_v1 as
         sex,
 
         -- Misc cruft
-        sample.details->>'sample_origin' as manifest_origin
+        sample.details->>'sample_origin' as manifest_origin,
+        sample.details->>'swab_type' as swab_type,
+        sample.details->>'collection_matrix' as collection_matrix
```

Comment:
Hah I just realized these are under Tom's comment calling this "Misc cruft". I don't think swab type and collection matrix fall under "cruft". Might be worth moving those details above the comment or removing the comment altogether.

Reply:
Pushing new commit to update to `--sample details columns`. I'm not sure it's that informative, but along wIth the other comments in this view, it at least allows for logical grouping of variables and easier scanning.